runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineDebugOptions.java

```diff
@@ -192,6 +192,22 @@ public Dataflow create(PipelineOptions options) {
 
   void setDumpHeapOnOOM(boolean dumpHeapBeforeExit);
 
+  /**
+   * The GC thrashing threshold (0.00 - 100.00) for every period to be used by MemoryMonitor. If the
+   * time spent on garbage collection in one period exceeds this threshold, that period is
+   * considered to be in GC thrashing.
+   *
+   * <p>If {@literal 100} is given as the value, MemoryMonitor will be disabled.
+   */
+  @Description(
+      "The GC thrashing threshold (0.00 - 100.00) for every period to be used by MemoryMonitor. "
```

Comment:
Avoid confusion with minus sign, and phrase more in terms of behavior. Maybe something like:

> The GC thrashing threshold percentage. A given period of time is considered "thrashing" if this percentage of CPU time is spent in garbage collection. Dataflow will force fail tasks after sustained periods of thrashing."

You could say "percentage in the range (0.0, 100.0)" but I think that is redundant.